lib/UtapiReindex.js

```diff
@@ -81,15 +81,34 @@ class UtapiReindex {
         return this.ds.del(REINDEX_LOCK_KEY);
     }
 
+    _buildFlags() {
+        const flags = {
+            /* eslint-disable camelcase */
+            sentinel_ip: this._sentinel.host,
+            sentinel_port: this._sentinel.port,
+            sentinel_cluster_name: this._sentinel.name,
+            bucketd_addr: `http://${this._bucketd.host}:${this._bucketd.port}`,
+        };
+        if (this._sentinel.sentinelPassword) {
+            flags.redis_password = this._sentinel.sentinelPassword;
+        }
+        /* eslint-enable camelcase */
+        const opts = [];
+        Object.keys(flags)
```

Comment:
I am still new here but this forEach looks like a good candidate for the Object.entries method since you reference both the key and value. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries.

Reply:
I didn't know about `entries` and it seems bert-e beat me and merged. Thanks for the tip though!

Reply:
I didn't know about entries and it seems bert-e beat me and merged. Thanks for the tip though!

Reply:
No worries, glad I could help.